README.md

```diff
@@ -3,6 +3,48 @@
 A [Singer](https://singer.io) Tap to extract currency exchange rate
 data from [fixer.io](http://fixer.io).
 
+## How to use it
+
+### Install and Run
+
+First, make sure Python 3 is installed on your system or follow these
+installation instructions for [Mac] or [Ubuntu].
+
+First, get an access key from [fixer.io](http://fixer.io). You need at least
+the basic plan to get exchange rates from a base currency (such as USD).
+
+Convert `config.sample.json` to `~/singer.io/tap_fixerio_config.json`; fill out your parameters.
+
+It's recommended to use a virtualenv:
+
+```bash
+python3 -m venv ~/.virtualenvs/tap-fixerio
+source ~/.virtualenvs/tap-fixerio/bin/activate
+pip install -U pip setuptools
+pip install -e '.'
+```
+
+Set up the `target-csv` virtual environment according to the instructions
+[here](https://github.com/singer-io/target-csv/blob/master/README.md).
+These commands will install `tap-fixerio`  with pip, and then run it:
+
+```bash
+~/.virtualenvs/tap-fixerio/bin/tap-fixerio --config ~/singer.io/tap_fixerio_config.json
+```
+
+The data will be written to a file called `exchange_rate.csv` in your
+working directory.
+
+$ cat exchange_rate.csv
```

Comment:
I believe @timvisher  suggested I remove them since it's not code- it's plain text.

Reply:
I believe he was just talking about the `>` and the `bash`. The backticks should have been left untouched so that the markdown file renders the log in a different font.